_layouts/coe.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+---
+
+<section class="usa-grid usa-section">
+    <h1>{{ page.title }}</h1>
+    <div class="usa-width-two-thirds">
+        <p class="intro">{{ page.tagline }}</p>
+
+        <div class="page-nav">
+            <a href="#service offerings">Service Offerings</a> | {% if page.service-catalog %}<a href="#service catalog">Service Catalog</a> | {% endif %}<a href="#coe-updates">Latest Updates</a> | {% for item in page.page-nav.children %}<a href="{{ site.baseurl }}/work/{{ item.url-fragment }}">Our Work at {{ item.agency }}</a> {% if forloop.last == false %} | {% endif %}{% endfor %}
+        </div>
+
+      <p>{{ page.intro }}</p>
+
+      <h2 id="service-offerings">Service Offerings</h2>
+      <ul>
+      {% for offering in page.service-offerings %}
+        <li>{{ offering }}</li>
+      {% endfor %}
+      </ul>
+
+      {% if page.service-catalog %}
```

Comment:
Not every page has a service catalog section, so only render it if it exists in the front matter.